inst/extdata/reportingVariables/Mapping_EuropeanEnergyDatasheets.csv

```diff
@@ -212,9 +226,11 @@
 "Energy Balance|Final energy consumption|Renewables and biofuels|Ambient heat, heat pumps";"Mtoe";"FE|Heat|Heat Pumps (EJ/yr)";0.041868
 "Energy Balance|Final energy consumption|Waste, non-renewable";"Mtoe";"FE|Solids (EJ/yr)";0.041868
 "Energy Balance|Final energy consumption|Waste, non-renewable";"Mtoe";"FE|Solids|Waste (EJ/yr)";0.041868
+"Energy Balance|Final energy consumption|Waste, non-renewable";"Mtoe";"SE|Solids (EJ/yr)";0.041868
```

Comment:
sorry, can you add two additional lines?
"Energy Balance|Final energy consumption|Waste, non-renewable";"Mtoe";"FE|Solids|Fossil (EJ/yr)";0.041868
"Energy Balance|Final energy consumption|Waste, non-renewable";"Mtoe";"SE|Solids|Fossil (EJ/yr)";0.041868